src/helpers/snapshot.ts

```diff
@@ -103,7 +103,7 @@ export async function getPower(space, address, proposal) {
     let scores: any = await getScores(
       space.id,
       strategies,
-      space.network,
+      proposal.network || space.network,
```

Comment:
We should use only proposal.network everywhere (without space network fallback), all proposals have a network

Reply:
@bonustrack Done